src/squidpy/tl/_sliding_window.py

```diff
@@ -33,36 +40,49 @@ def sliding_window(
     Parameters
     ----------
     %(adata)s
-    window_size: int
-        Size of the sliding window.
     %(library_key)s
     coord_columns: Tuple[str, str]
         Tuple of column names in `adata.obs` that specify the coordinates (x, y), e.i. ('globalX', 'globalY')
+    window_size: int
+        Size of the sliding window.
+    %(spatial_key)s
     sliding_window_key: str
         Base name for sliding window columns.
-    overlap: int
-        Overlap size between consecutive windows. (0 = no overlap)
-    %(spatial_key)s
     drop_partial_windows: bool
         If True, drop windows that are smaller than the window size at the borders.
+    overlap: int
+        Overlap size between consecutive windows. (0 = no overlap)
+    max_n_cells: int
+        If window_size is None, either 'n_split' or 'max_n_cells' can be set.
+        max_n_cells sets an upper limit for the number of cells within each region.
+    n_splits: int
+        This can be used to split the entire region to some splits.
     copy: bool
         If True, return the result, otherwise save it to the adata object.
+    split_line: str
+        If 'square' is False, this set's the orientation for rectanglular regions. `h` : Horizontal, `v`: Vertical
 
     Returns
     -------
     If ``copy = True``, returns the sliding window annotation(s) as pandas dataframe
     Otherwise, stores the sliding window annotation(s) in .obs.
     """
+
     if overlap < 0:
         raise ValueError("Overlap must be non-negative.")
 
     if isinstance(adata, SpatialData):
         adata = adata.table
 
+    assert max_n_cells is None or n_splits is None, (
```

Comment:
Also: assert that not all `window_size`, `max_n_cells` and `n_splits` are `None`.